app/page.tsx

```
@@ -1,137 +1,74 @@
+      <section className="py-16 bg-primary text-primary-foreground">
+        <ResponsiveContainer maxWidth="xl" className="text-center">
+          <ResponsiveStack direction="vertical" spacing="6" className="max-w-3xl mx-auto">
+            <h2 className="text-3xl font-bold tracking-tight">Are You a Provider?</h2>
+            <p className="text-xl">
+              Join our platform to showcase your services and connect with potential clients
+            </p>
+            <div>
+              <a 
+                href="/sign-up" 
+                className="inline-flex items-center justify-center rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 bg-primary-foreground text-primary hover:bg-primary-foreground/90 h-10 px-4 py-2"
               >
```

ESLint

[error] Do not use an `<a>` element to navigate to `/sign-up/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)